.github/workflows/linux.yml

```diff
@@ -1357,6 +1357,91 @@ jobs:
             ${{ env.INSTALL_TEST_DIR }}/TEST*.html
           if-no-files-found: 'error'
 
+  TensorFlow_Hub_Performance_Models_Tests:
+    name: TensorFlow Hub Performance Models tests
+    needs: Build
```

Comment:
you also need to depend on the smart ci job

Reply:
Done